app/controllers/account_home_controller.rb

```diff
@@ -39,4 +40,15 @@ def confirmed_user_changed_email?
 
     @user["email_verified"] && @user["has_unconfirmed_email"]
   end
+
+  def has_email_subscriptions?
+    subscriber = GdsApi.email_alert_api.authenticate_subscriber_by_govuk_account(
+      govuk_account_session: @account_session_header,
+    ).to_h.fetch("subscriber")
+
+    subscriptions = GdsApi.email_alert_api.get_subscriptions(id: subscriber.fetch("id")).to_h.fetch("subscriptions", [])
+    !subscriptions.empty?
```

Comment:
Design question: do we want to show the "you have subscriptions"-state service card to *all* users with subscriptions, or only those who have also linked their notifications to their account?  You can check that by seeing if `subscriber["govuk_account_id"]` is present.

Reply:
Checked with the team and we want to show this panel to anyone with a subscription